src/ngraph/serializer.cpp

```diff
@@ -453,7 +462,7 @@ static string serialize(shared_ptr<Function> func, size_t indent, bool binary_co
 
 std::string ngraph::serialize(std::shared_ptr<ngraph::Function> func, size_t indent)
 {
-    return ::serialize(func, indent, false);
+    return ::serialize_to_json(func, indent, false);
```

Comment:
function name is `serialize` but in body we have only serialization to json is it intended?
Maybe we should change this function name.

Reply:
As a element of unification I added possibility to call serialize method with expected cpio output type for this overloaded ```serialize``` method